bin/woff2_compress.js

```diff
@@ -17,23 +17,34 @@ var parser = new argparse.ArgumentParser({
   addHelp:  true
 });
 
-parser.addArgument([ 'infile' ],  { nargs: 1, help: 'Input file' });
-parser.addArgument([ 'outfile' ], { nargs: 1, help: 'Output file' });
+parser.addArgument([ 'infile' ],  { nargs: 1, help: 'Input .ttf file' });
+parser.addArgument([ 'outfile' ], { nargs: '?', help: 'Output .woff2 file (- for stdout)' });
 
 ////////////////////////////////////////////////////////////////////////////////
 
 let args = parser.parseArgs();
+let infile = args.infile[0];
+let outfile = args.outfile;
```

Comment:
`args.outfile` is not an array as `nargs` is now equal to `?`. It's null or something.